### PR TITLE
(docker)resolve Dockerfile issue & fix CVEs

### PR DESCRIPTION
**Pull request by huynaism**
Remaining CVEs :
| Lib      | CVE                                                                      | Current version | Fixed version | Indirect |
| -------- | ------------------------------------------------------------------------ | --------------- | ------------- | -------- |
| CometBFT | [GHSA-p7mv-53f2-4cwj](https://github.com/advisories/GHSA-p7mv-53f2-4cwj) | v0.38.14        | v0.38.15      |          |
| wasmd    | [GHSA-g8w7-7vgg-x7xg](https://github.com/advisories/GHSA-g8w7-7vgg-x7xg) | 0.52.0          | 0.53.0        |          |
| wasmd    | [GHSA-fpgj-cr28-fvpx](https://github.com/advisories/GHSA-fpgj-cr28-fvpx) | 0.52.0          | 0.53.0        |          |

### CODE REVIEW

**Review by Lazar955 — APPROVED** [review]
Nice work! Update the changelog

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ on:
     branches:
     - 'main'
     - 'dev'
+    - 'devops/add-docker-sec'
```
Comment (Lazar955) [inline .github/workflows/publish.yml:8]: is this needed?

Comment (huynaism) [reply .github/workflows/publish.yml]: actually not! removed it in https://github.com/babylonlabs-io/babylon/pull/264/commits/b627a675654f77b80a183e87c31cdbcee597d993

**Comment by huynaism** [comment]
> Nice work! Update the changelog

Thanks! Changelog updated

**Pull request merged**